environments/cvms.md

```diff
@@ -18,7 +18,7 @@ a _Container-based Virtual Machine (CVM)_.
 > [enable Docker in Environments](../admin/environment-management/cvms.md)
 > for your deployment.
 
-## Container-based Virtual Machine
+## Container-based Virtual Machine (CVMs)
```

Comment:
We should do either

> Container-based Virtual Machines (CVMs)

or

> Container-based Virtual Machine (CVM)

I'm inclined toward the singular

Reply:
Agreed. Updated to (CVM) in the newest commit.